src/pages/test-report/test-report.html

```diff
@@ -8,8 +8,8 @@
 </ion-header>
 
 <ion-content no-padding no-bounce>
-  <div style="background: white; height: 68px; padding: 0">
-    Temp Top Header
+  <div id="test-report-header" style="background: white; height: 68px; padding: 12px 17px 12px 17px;">
```

Comment:
You've extracted these out to the css class. So they can be removed from here

Reply:
removed